src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -32,6 +32,10 @@ class GraphvizDumper implements DumperInterface
         'edge' => array('fontsize' => 9, 'fontname' => 'Arial', 'color' => '#333333', 'arrowhead' => 'normal', 'arrowsize' => 0.5),
     );
 
+    private static $defaultAttributes = array(
+        'fixedsize' => 'false',
```

Comment:
we can simply update the default value above no?

https://github.com/symfony/symfony/blob/f27c3a85843f7bfbd08ebb6793b207a4f28c33f1/src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php#L31

Reply:
When typing "I tried that and it didn't work...", I realized the problem was the `true` (current value) translated to `fixedsize="1"`, so the value should not be a `boolean` but a string...
I changed the `true` into `'false'` and that seem to do the trick as well ;). Thanks!